website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -13,6 +13,16 @@ description: This guide walks through an example of building a simple memcached-
 - Access to a Kubernetes v1.11.3+ cluster (v1.16.0+ if using `apiextensions.k8s.io/v1` CRDs).
 - User logged with admin permission. See [how to grant yourself cluster-admin privileges or be logged in as admin][role-based-access-control]
 
+## Overview
+
+By using operators, it’s possible not only to provide all expected resources but also to manage them dynamically, programmatically, and at execution time. To illustrate this idea, imagine if someone accidentally changed a configuration or removed a resource by mistake; in this case, the operator could fix it without any human intervention. 
```

Comment:
Not really sure this is the best spot for this. This paragraph seems to be explaining why you would want to use an operator instead of some static solution like helm, but shouldn't it be explaining an overview of what the tutorial is about?

Reply:
good spot 👍

Reply:
removed